frontend/src/components/misc/KowlJsonView.tsx

```diff
@@ -133,15 +133,15 @@ export const KowlJsonView = observer((props: ReactJsonViewProps) => {
                         collapsed={settings.collapsed}
                         onSelect={e => {
                             if (ctrlDown) {
-                                if (navigator?.clipboard) {
-                                    navigator.clipboard.writeText(String(e.value));
-                                    message.success(
-                                        <span>
-                                            Copied value of <span className="codeBox">{e.name}</span>
+                                navigator.clipboard.writeText(String(e.value)).then(() => {
+                                    toast({
```

Comment:
should have `.catch(...` or checking if clipboard is available; there are cases where its not

Reply:
Fixed